_posts/2023-12-18-homebrew-4.2.0.md

```diff
@@ -0,0 +1,298 @@
+---
+title: 4.2.0
+author: MikeMcQuaid
+redirect_from: /blog/4.2.0/
+---
+
+Today, I'd like to announce Homebrew 4.2.0. The most significant changes since 4.1.0 are .
+
+Major changes and deprecations since 4.1.0:
+
+- Ruby 3
+- [Portable Ruby 3.1.4 (opt-in beta)](https://github.com/Homebrew/brew/pull/16145)
+- [Require Ruby 3.1](https://github.com/Homebrew/brew/pull/16294)
+- [Support system Ruby 3 on Linux](https://github.com/Homebrew/brew/pull/16192)
+
+- [Enable Ruby 3 for all users](https://github.com/Homebrew/brew/pull/16268)
+- [Deprecate `--new-formula`/`--new-cask` options.](https://github.com/Homebrew/brew/pull/16297)
+- [Deprecate `OS::Mac` on Linux](https://github.com/Homebrew/brew/pull/16224)
+- [Deprecate `postgresql-upgrade-database`](https://github.com/Homebrew/brew/pull/15799)
+- [Deprecate, disable & delete code for Homebrew 4.2.0](https://github.com/Homebrew/brew/pull/15632)
+
+Other changes since 4.1.0 I'd like to highlight are the following:
+
+- [Add M3 processors](https://github.com/Homebrew/brew/pull/16278)
+- [Add package signing configuration for GitHub Actions](https://github.com/Homebrew/brew/pull/15743)
+- [Allow configuring Homebrew with `.env` files](https://github.com/Homebrew/brew/pull/15787)
+- [Allow hiding new/deleted formula/cask sections on update](https://github.com/Homebrew/brew/pull/15836)
+- [Cask support multiple headers](https://github.com/Homebrew/brew/pull/15602)
```

Comment:
Likewise if we can rename this so that it's clear just from reading the changelog.